src/Item.h

```diff
@@ -206,6 +222,7 @@ class cItem
 
 	int            m_RepairCost;
 	cFireworkItem  m_FireworkItem;
+	unsigned int m_ItemColor;
```

Comment:
I took this approach because sending an item across the network will happen more frequently than changing an item color.  As it stands right now, I don't have to compress the color into an int every single time the item is sent to a client, and I can still alter the rgb values with minimal manipulation.